test/test_multiprocessing.py

```diff
@@ -1061,6 +1061,9 @@ def test_is_shared_cuda(self):
         t = torch.randn(5, 5).cuda()
         self.assertTrue(t.is_shared())
 
+    def test_set_thread_name(self):
+        mp._set_thread_name("test name")
```

Comment:
is there a way we could read this back to check it got set?

Reply:
done